src/month_delta.rs

```diff
@@ -0,0 +1,31 @@
+use polars::prelude::*;
+
+pub(crate) fn impl_month_delta(start_dates: &Series, end_dates: &Series) -> PolarsResult<Series> {
+    let start_dates = start_dates.date()?;
+    let end_dates = end_dates.date()?;
+
+    if (start_dates.dtype() != &DataType::Date) || (end_dates.dtype() != &DataType::Date) {
+        polars_bail!(InvalidOperation: "polars_xdt.workday_count only works on Date type. Please cast to Date first.");
```

Comment:
copy-pasta

Reply:
whoops, copied an excellent piece of code